lib/pages/full_post/full_post.dart

```diff
@@ -27,17 +27,32 @@ import 'full_post_store.dart';
 
 /// Displays a post with its comment section
 class FullPostPage extends HookWidget {
-  const FullPostPage._();
+  final bool toComments;
+
+  const FullPostPage({this.toComments = false});
 
   @override
   Widget build(BuildContext context) {
     final scrollController = useScrollController();
     final shareButtonKey = GlobalKey();
+    final postAnchor = GlobalKey();
     var scrollOffset = 0.0;
 
     final loggedInAction =
         useLoggedInAction(context.read<FullPostStore>().instanceHost);
 
+    if (toComments) {
+      WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
I also think this will run on every update to this widget, since the tree needs to rebuild.
If you use the `useEffect()` hook, you can give `[toComments]` as the second argument. Then it will only run if that value changes (which it probably won't).